### DISCUSSION TITLE
Laravel 11 compatibility

### DISCUSSION THREAD

**Discussion by TSchuermans** [spatie/pixelmatch-php#10] [Q&A] [answered]
Would it be possible to tag a release?
This commit from 4 months ago would fix L11 compatibility.
https://github.com/spatie/pixelmatch-php/commit/437b6a630ea9f5d2cd831bab89e01b2e46a5885b

**Comment by riasvdv** [comment, marked as answer]
I've tagged a new release 👍

**Comment by TSchuermans** [comment]
Thanks @riasvdv !
